app/helpers/application_helper.rb

```diff
@@ -13,4 +13,12 @@ def flash_class(level)
       'bg-blue-900 border-blue-900'
     end
   end
+
+  def job_options_for_select(account_id)
+    Job.where(account_id: account_id).order(:title).pluck(:title, :id)
+  end
+
+  def fetch_filter_key(resource, user_id, key)
+    Kredis.hash("#{resource}_filters:#{user_id}")[key]
```

Comment:
Usage of the `"#{resource.to_s.underscore}_filters:#{user_id}"` key might be more consistent with the `Filterable` concern. In that case we can either write `fetch_filter_key(Applicant, current_user.id, "sort")` or `fetch_filter_key("applicant", current_user.id, "sort")`